rust_src/src/math.rs

```diff
@@ -291,3 +291,101 @@ fn abs(arg: LispObject) -> LispObject {
         }
     }
 }
+
+#[repr(C)]
+#[derive(Clone, Copy)]
+pub enum ArithComparison {
+    ArithEqual,
```

Comment:
In Rust enums, the prefix for the values is usually omitted since you fully qualify the names. (I.e. `ArithComparison::Equal`.)